src/api/objects/ObjectAPI.js

```diff
@@ -312,6 +319,35 @@ define([
         });
     };
 
+    /**
+     * Register a new object interceptor.
+     *
+     * @param {string} key a string identifier for this interceptor
+     * @param {module:openmct.Interceptor} interceptor the interceptor to add
+     * @method addInterceptor
+     * @memberof module:openmct.InterceptorRegistry#
+     */
+    ObjectAPI.prototype.addInterceptor = function (key, interceptorDef) {
```

Comment:
Let's name this `addGetInterceptor`. We might also want to add `save` interceptors later for eg.

Reply:
How do envision separating the get and save interceptors? Would they just be different objects on this class? That would work and then we wouldn't need keys either.

Reply:
Done

Reply:
Yeah, I guess they'd be separate methods? `addGetInterceptor`, `addSaveInterceptor`.